content/docs/examples/advanced-gateways/ingress-certmgr/index.md

```diff
@@ -0,0 +1,174 @@
+---
+title: Securing Kubernetes Ingress with Cert-Manager
+description: Demonstrates how to obtain "Let's Encrypt" TLS certificates for Kubernetes Ingress automatically using Cert-Manager.
+weight: 70
+keywords: [traffic-management,ingress,https,cert-manager,acme,sds]
+---
+
+This tutorial demonstrates the use of Istio as a secure Kubernetes Ingress controller with TLS certificates issued by "Let's Encrypt". We will start with a clean Istio installation, create an example service, expose it using the Kubernetes `Ingress` resource and get it secured by instructing `cert-manager` (bundled with Istio) to manage issuance and renewal of TLS certificates that will be further delivered to the Istio Ingress Gateway and hot-swapped as necessary via the means of SDS (Secrets Discovery Service).
+
+## Before you begin
+
+Install Istio by following the instructions corresponding to your platform [installation guide](/docs/setup/). Make sure to enable Ingress Gateway with Kubernetes Ingress support, SDS and [cert-manager](https://docs.cert-manager.io/) optional dependency during installation. Here's an example of how to do this for the [`helm template`](/docs/setup/kubernetes/install/helm/#option-1-install-with-helm-via-helm-template) installation path:
```

Comment:
I think this is `(Instlalation Guide)[/docs/setuep] but could be mistaken.

`helm template`  should not be in ` as it is in a link.

Reply:
Copied this exact link from other examples (i.e. bookinfo) -- exact same formatting.

Reply:
...and removed backticks around 'helm template'